operations/registers.md

```diff
@@ -0,0 +1,115 @@
+<!-- AUTOMATICALLY GENERATED, DO NOT MODIFY -->
+# Registers
+
+## Get registers
+
+A register represents a terminal or cash register within an outlet, used to issue invoices. Each register is uniquely identified within its outlet and can either be physical or virtual.
+Note: This operation needs [Authentication](../essential-guide/authentication.md) and supports the `include` query parameter `outlet` following the [JSON:API specification](../essential-guide/features.md).
+
+### Request
+
+`GET` `[PlatformAddress]/api/v2/registers/{id}`
+
+### Response
+
+```javascript
+{
+  "data": {
+    "id": "31b14937-2524-491f-b0a0-dc0a7393ff3f",
+    "type": "registers",
+    "attributes": {
+      "name": "Main Register",
+      "invoices_count": 1200,
+      "index": 3,
+      "virtual": false,
+      "created_at": "2023-10-16T14:30:00Z",
+      "updated_at": "2023-10-19T14:30:00Z"
+    },
+    "relationships": {
+      "outlet": {
+        "data": {
+          "id": "5efa8b3c-b930-4b31-918d-95ab0e212e64",
+          "type": "outlets"
+        }
+      }
+    }
+  },
+  "included": [
+    {
+      "id": "5efa8b3c-b930-4b31-918d-95ab0e212e64",
+      "type": "outlets",
+      "attributes": {
+        "name": "Downtown Cafe",
+        "address1": "123 Main Street",
+        "address2": "Suite 4B",
+        "city": "New York",
+        "State": "New York",
+        "postal_code": "10001",
+        "index": 5,
+        "created_at": "2023-10-16T13:30:00Z",
+        "updated_at": "2023-10-16T15:30:00Z"
+      }
+    }
+  ]
+}
+```
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `data` | [register](registers.md#register) | required | The document's "primary data". |
+| `included` | array of object [outlet](registers.md#outlet) | optional, max 1 item | Details of the outlet to which the register is associated. |
+
+#### register
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `id` | string | required, max length 36 characters | Universally unique ID (UUID) that identifies the related object. |
```

Comment:
GUIDs are Microsoft's implementation of UUIDs. Although practically they are identical, however, I felt that it would be more accurate to note that we are using UUIDs.